src/components/comment-list-item.component.js

```diff
@@ -13,71 +13,71 @@ import moment from 'moment/min/moment-with-locales.min';
 import { translate } from 'utils';
 import { colors, fonts, normalize } from 'config';
 
-const styles = StyleSheet.create({
-  container: {
-    paddingRight: 10,
-    paddingTop: 10,
-    backgroundColor: 'transparent',
-  },
-  header: {
-    flexDirection: 'row',
-    marginLeft: 10,
-    alignItems: 'center',
-  },
-  avatarContainer: {
-    backgroundColor: colors.greyLight,
-    borderRadius: 17,
-    width: 34,
-    height: 34,
-  },
-  avatar: {
-    width: 34,
-    height: 34,
-    borderRadius: 17,
-  },
-  titleSubtitleContainer: {
-    justifyContent: 'center',
-    flex: 1,
-    marginLeft: 10,
-  },
-  dateContainer: {
-    flex: 0.15,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-  linkDescription: {
-    ...fonts.fontPrimaryBold,
-    fontSize: normalize(13),
-    color: colors.primaryDark,
-  },
-  date: {
-    color: colors.greyDark,
-  },
-  commentContainer: {
-    paddingTop: 5,
-    marginLeft: 54,
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  commentBottomPadding: {
-    paddingBottom: 15,
-  },
-  commentText: {
-    fontSize: normalize(12),
-    color: colors.primaryDark,
-  },
-  commentTextNone: {
-    ...fonts.fontPrimary,
-    color: colors.primaryDark,
-    fontStyle: 'italic',
-  },
-  actionButtonIconContainer: {
-    paddingTop: 5,
-    paddingBottom: 10,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-});
+const Container = styled.View`
+  padding: 10px 10px 0 0;
+  background-color: transparent;
+`;
+
+const Header = styled.View`
+  flex-direction: row;
+  margin-left: 10px;
+  align-items: center;
+`;
+
+const AvatarContainer = styled.TouchableOpacity`
+  background-color: ${colors.greyLight};
+  border-radius: 17px;
+  width: 34px;
+  height: 34px;
+`;
+
+const Avatar = styled.Image`
+  border-radius: 17px;
+  width: 34px;
+  height: 34px;
+`;
+
+const TitleSubtitleContainer = styled.View`
+  justify-content: center;
+  flex: 1;
+  margin-left: 10px;
+`;
+
+const DateContainer = styled.View`
+  flex: 0.1;
+  align-items: flex-end;
+  justify-content: center;
+`;
+
+const LinkDescription = styled.Text`
+  ${{ ...fonts.fontPrimaryBold }};
```

Comment:
Can we just write as `${fonts.fontPrimaryBold}`?

Reply:
I don’t kno. I’m not sure how this actually works. Just copied it from the other already concerted components. I handle fonts differently in my apps.

Reply:
This should be:

`font-family: ${styledFonts.fontPrimaryBold}`